rstudio/rstudio
# Fix Run Tests button in non-project sessions

### Intent

Addresses https://github.com/rstudio/rstudio/issues/8775. 

### Approach

This change just reverts https://github.com/rstudio/rstudio/pull/7756, which removed the Tests pane. 

### Automated Tests

None, this change restores the product to a previous state.

### QA Notes

There are 3 main cases to consider here:

1. Sessions in which projects are not open
2. Project sessions without a Build pane (e.g. New Empty Project)
3. Project sessions with a Build pane (e.g. New Package Project)

The Tests pane should show up in cases (1) and (2). In case (3), tests should be run in the Build pane instead.

### Checklist
